core/domain/improvements_domain_test.py

```diff
@@ -89,7 +88,7 @@ def test_to_dict_has_expected_value(self) -> None:
             'status': 'resolved',
             'resolver_username': self.OWNER_USERNAME,
             'resolver_profile_picture_data_url': (
-                user_services.DEFAULT_IDENTICON_DATA_URL),
+                default_identicon_data_url),
```

Comment:
can be move to previous line

Reply:
Will be now `None` due to changes in `to_dict()`.